diff --git a/app/js/init.js b/app/js/init.js
index 68f01a0..3b4f3d4 100644
--- a/app/js/init.js
+++ b/app/js/init.js
@@ -1,11 +1,7 @@
 ;(function initApplication () {
   // Prevent click-jacking
   try {
-    if (window == window.top || window.chrome && chrome.app && chrome.app.window) {
       document.documentElement.style.display = 'block'
-    } else {
-      top.location = self.location
-    }
   } catch (e) {console.error('CJ protection', e) }
 
   var classes = [
diff --git a/package.json b/package.json
index 358ea16..f660786 100644
--- a/package.json
+++ b/package.json
@@ -20,7 +20,8 @@
   },
   "scripts": {
     "start": "gulp watch",
-    "test": "gulp test"
+    "test": "gulp test",
+    "build": "gulp publish"
   },
   "repository": {
     "type": "git",
